### PR TITLE
add git page

### PR DESCRIPTION
**Pull request by lifecoder1988**

### CODE REVIEW

**Comment by barnjamin** [comment]
Thanks @lifecoder1988 , can you please provide a summary of this change?

@fabrice102 can you preview this change?

**Comment by michielmulders** [comment]
@ori-shem-tov tagging Ori as well

**Comment by fabrice102** [comment]
@lifecoder1988 Thanks a lot! In addition to @barnjamin's question, can you give us an easy way to preview the page, i.e.:
* add the necessary Gemfiles and write the instructions to test it locally (like https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
* ideally publish it in your github pages, and give the link (but not absolutely necessary)

**Comment by lifecoder1988** [comment]
> @lifecoder1988 Thanks a lot! In addition to @barnjamin's question, can you give us an easy way to preview the page, i.e.:
> 
> * add the necessary Gemfiles and write the instructions to test it locally (like https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll)
> * ideally publish it in your github pages, and give the link (but not absolutely necessary)

gemfile has been added .

use this command to host locally , the output html will be generate in _site dir.

` bundle exec jekyll serve`

**Comment by ori-shem-tov** [comment]
Thanks @lifecoder1988 !

All in all looking good, just a few remarks:
- In the `Last Call` section the title of the ARC is in the `Review ends` column. Can you please fix that and make there shouldn't be any more issues like that with other sections?
- In the `Draft` section, there's no number for the ARC with title `Algorand standard for declaring traits in an ASA's metadata`
- The footer has overlapping text in it:
image

**Comment by lifecoder1988** [comment]
> Thanks @lifecoder1988 Also, please remove any `Ethereum`, `EIP` or any other related. In addition, there's no link to the discussion (found in the the github issues section) for each ARC. Can you please add that as well?
hi @ori-shem-tov 
if you can add link to each ARC file , then i can add it . otherwise, i can't .

**Comment by ori-shem-tov** [comment]
Hi @lifecoder1988,
One option for linking an ARC to its  discussion is adding another column to the ARC's header (should be also added to the ARC template) and insert the URL to the corresponding issue there. You can find all the discussion here: https://github.com/algorandfoundation/ARCs/issues?q=is%3Aopen+is%3Aissue

**Comment by lifecoder1988** [comment]
> Hi @lifecoder1988, One option for linking an ARC to its discussion is adding another column to the ARC's header (should be also added to the ARC template) and insert the URL to the corresponding issue there. You can find all the discussion here: https://github.com/algorandfoundation/ARCs/issues?q=is%3Aopen+is%3Aissue

hi @ori-shem-tov 
it's already supported . you can change ARC File (MarkDown format) to get it work . 

image

image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <li><strong>Last Call</strong> - This is the final review window for an ARC before moving to FINAL. An ARC editor will assign Last Call status and set a review end date (`last-call-deadline`), typically 14 days later. If this period results in necessary normative changes it will revert the ARC to Review.</li>
+  <li><strong>Final</strong> - This ARC represents the final standard. A Final ARC exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.</li>
+  <li><strong>Stagnant</strong> - Any ARC in Draft or Review if inactive for a period of 6 months or greater is moved to Stagnant. An ARC may be resurrected from this state by Authors or ARC Editors through moving it back to Draft.</li>
+  <li><strong>Withdrawn</strong> - The ARC Author(s) have withdrawn the proposed ARC. This state has finality and can no longer be resurrected using this ARC number. If the idea is pursued at later date it is considered a new proposal.</li>
+  <li><strong>Living</strong> - A special status for ARCs that are designed to be continually updated and not reach a state of finality. This includes most notably ARC-1.</li>
```
Comment (ori-shem-tov) [inline index.html:22]:
```suggestion
  <li><strong>Living</strong> - A special status for ARCs that are designed to be continually updated and not reach a state of finality.</li>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Home
+---
+
+<h1 class="page-heading">ARCs
+  <a href="#"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
+  <a href="#"><img src="https://img.shields.io/badge/rss-Last Calls-red.svg" alt="RSS"></a>
```
Comment (ori-shem-tov) [inline index.html:8]:
```suggestion
  <a href="https://discord.com/invite/algorand"><img src="https://img.shields.io/discord/491256308461207573?style=flat" alt="Discord"></a>
```

**Pull request merged**